### PR TITLE
[MLEPNG-1322] Update requirements.txt

### PR DESCRIPTION
**Pull request by ludens11**
## Purpose
To update `requirements.txt` with additional item:
- Adding standard data science libraries
- Adding `--extra-index-url` to install from Traveloka private artifact registries

### CODE REVIEW

File: default/{{ cookiecutter.model_name }}-trainer/requirements.txt [hunk 1]
```diff
@@ -1,2 +1,9 @@
-kfp==1.6.2
-https://storage.googleapis.com/public-python-wheels/rm-trainer-sdk/releases/latest/rm_trainer_sdk-stable-py3-none-any.whl
\ No newline at end of file
+scipy
+scikit-learn
+pandas
+google-cloud-storage
+pyyaml
```
Comment (devina-ekawati) [inline default/{{ cookiecutter.model_name }}-trainer/requirements.txt:5]: should we pin the version? 🤔

Comment (ludens11) [reply default/{{ cookiecutter.model_name }}-trainer/requirements.txt:5]: oh yes, we should, any suggestion to which version should we use?

Comment (ludens11) [reply default/{{ cookiecutter.model_name }}-trainer/requirements.txt:5]: maybe we can use the latest one for now? 🤔

**Review by devina-ekawati — APPROVED** [review]
lgtm

**Pull request merged**